library/network/src/lib/y2firewall/firewalld/api/zones.rb

```diff
@@ -69,19 +85,51 @@ def list_sources(zone, permanent: permanent?)
           string_command("--zone=#{zone}", "--list-sources", permanent: permanent).split(" ")
         end
 
+        # @param zone [String] The firewall zone
+        # @param permanent [Boolean] if true it adds the --permanent option the
+        # command to be executed
```

Comment:
Please, indent this line. Otherwise, the documentation won't be rendered correctly (same for all the "command to be executed" lines in this file).